src/app/utils/BountyUtils.ts

```diff
@@ -83,6 +84,15 @@ const BountyUtils = {
         }
     },
 
+    validateEvergreen(evergreen: boolean, claimLimit: number) {
+        if (!evergreen && claimLimit !== undefined) {
+            throw new ValidationError('claim-limit is only used for evergreen bounties.');
+        }
+        if (claimLimit !== undefined && (claimLimit < 1 || claimLimit > 100)) {
+            throw new ValidationError('claim-limit should be from 1 to 100');
```

Comment:
upper limit of 100 is super sensible, would probably push the lower limit to 2

the evergreen flow has a learning curve in addition to regular bounties, and that feature is more to enable our power users. if a new user is coming in and selecting a limit of 1, we should push them to not use the evergreen feature

Reply:
Done